src/Nethermind/Nethermind.Consensus/Processing/ProcessingStats.cs

```diff
@@ -136,7 +136,7 @@ public void UpdateStats(Block? block, IBlockTree blockTreeCtx, int recoveryQueue
                             < 1000 => orangeText,
                             _ => redText
                         };
-                        _logger.Info($"Processed     {block.Number,9}           | {chunkColor}{chunkMs,9:N2}{resetColor} ms  | slot {runMs,7:N0} ms     |{blockGas}");
+                        _logger.Info($"Processed            {block.Number,9}    | {chunkColor}{chunkMs,9:N2}{resetColor} ms  | slot {runMs,7:N0} ms     |{blockGas}");
```

Comment:
why whitespace and not `\t` for example?

Reply:
Column alignment in console for readability; tab has no specific alignment? (align decimals, align numbers, align hash segments)
![image](https://github.com/NethermindEth/nethermind/assets/1142958/daa01e29-7e6a-4917-a029-3f952d93ff63)